test/test_nn.py

```diff
@@ -6480,14 +6480,14 @@ def test_RNN_change_dropout(self):
                 prev_output = output1.data
 
     def _verify_pixel_shuffle(self, input, output, upscale_factor):
```

Comment:
Add a comment explaining what this function does. If it's only used inside `_test_pixel_shuffle_helper` then it can be moved inside that function.